vtds_application_openchami/private/templates/quadlet/prepare_node.sh

```diff
@@ -61,8 +61,18 @@ dnf -y install ${PACKAGES}
 dnf -y install epel-release
 dnf -y install s3cmd
 systemctl enable --now libvirtd
-groupadd rocky
-useradd -g rocky -G libvirt rocky
+if ! grep "^rocky:" /etc/group; then
+    groupadd rocky
+fi
+if ! grep "^rocky:" /etc/passwd; then
```

Comment:
Usually something like `getent passwd rocky` is considered cleaner and safer alternative to direct querying of `/etc/passwd`.

Reply:
I fixed this and also added the same (fixed) logic to the bare system preparation as well. Thanks for the suggestion, that is much cleaner.